core/primitives/src/epoch_manager.rs

```diff
@@ -297,6 +299,12 @@ impl AllEpochConfig {
         }
     }
 
+    fn config_fix_min_stake_ratio(config: &mut EpochConfig, protocol_version: u32) {
+        if checked_feature!("stable", FixMinStakeRatio, protocol_version) {
+            config.validator_selection_config.minimum_stake_ratio = Rational32::new(1, 62_500);
+        }
+    }
+
```

Comment:
I don't have the knowledge to verify that it's the correct way to do it, but I assume that you know what you're doing. It's weird that we modify both the json file and the code?

Reply:
We are in the middle of changing source data from genesis config to small json configs, so for now both modifications are required.